src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -919,6 +919,14 @@ def load_arguments(self, _):
             help="Space-separated machine names to delete.",
         )
 
+    with self.argument_context("aks nodepool delete") as c:
+        c.argument(
+            "ignore_pod_disruption_budget",
+            options_list=["--ignore-pdb"],
```

Comment:
```suggestion
            options_list=["--ignore-pdb", "--ignore-pod-disruption-budget"],
```
please also include the default option name

Reply:
Added back, hope that pylint not to report length too long error.